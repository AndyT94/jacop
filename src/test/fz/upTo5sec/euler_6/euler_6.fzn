var 338350..338350: ss:: output_var;
var 5050..5050: s:: output_var;
solve  satisfy;
