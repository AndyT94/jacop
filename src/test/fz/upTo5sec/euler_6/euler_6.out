ss = 338350;
s = 5050;
----------